test/hotspot/jtreg/runtime/Monitor/MonitorWithDeadObjectTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
nit: why include 2022 in the copyright header?

Reply:
See earlier comment.